src/main/java/net/dv8tion/jda/core/EmbedBuilder.java

```diff
@@ -302,7 +301,7 @@ public StringBuilder getDescriptionBuilder()
      *
      * @return the builder after the description has been set
      */
-    public EmbedBuilder setDescription(CharSequence description)
+    public final EmbedBuilder setDescription(CharSequence description)
```

Comment:
Why was this needed?

Reply:
It's being called in the constructor, so it shouldn't be overridable

Reply:
But the overridden method would then be called which would be the intention of the override, no?